app/src/main/java/org/schabi/newpipe/player/helper/PlayerHelper.java

```diff
@@ -95,6 +101,19 @@ public final class PlayerHelper {
         int MINIMIZE_ON_EXIT_MODE_POPUP = 2;
     }
 
+    @Retention(SOURCE)
+    @IntDef({BACK_BEHAVIOR_PREV_ANY, BACK_BEHAVIOR_PREV_PLAYLIST,
+            BACK_BEHAVIOR_HIDE, BACK_BEHAVIOR_HIDE_OR_CLOSE,
+            BACK_BEHAVIOR_CLOSE, BACK_BEHAVIOR_CLOSE_FULLSCREEN})
+    public @interface BackBehavior {
+        int BACK_BEHAVIOR_PREV_ANY = 0;
+        int BACK_BEHAVIOR_PREV_PLAYLIST = 1;
+        int BACK_BEHAVIOR_HIDE = 2;
+        int BACK_BEHAVIOR_HIDE_OR_CLOSE = 3;
+        int BACK_BEHAVIOR_CLOSE = 4;
+        int BACK_BEHAVIOR_CLOSE_FULLSCREEN = 5;
+    }
```

Comment:
I wouldn't name all values inside "BackBehavior" with the prefix "BACK_BEHAVIOR_" because it's redundant.

Reply:
ALL interfaces in PlayerHelper are prefixed with their name. It's just made the same as the rest of the code